dcrec/secp256k1/btcec.go

```diff
@@ -679,14 +679,14 @@ func (curve *KoblitzCurve) moduloReduce(k []byte) []byte {
 	return k
 }
 
-// NAF takes a positive integer k and returns the Non-Adjacent Form (NAF) as two
+// naf takes a positive integer k and returns the Non-Adjacent Form (NAF) as two
 // byte slices.  The first is where 1s will be.  The second is where -1s will
 // be.  NAF is convenient in that on average, only 1/3rd of its values are
 // non-zero.  This is algorithm 3.30 from [GECC].
 //
 // Essentially, this makes it possible to minimize the number of operations
 // since the resulting ints returned will be at least 50% 0s.
-func NAF(k []byte) ([]byte, []byte) {
+func naf(k []byte) ([]byte, []byte) {
```

Comment:
Please don't unexport this.  First, it can be useful for external consumers of the module, but, even more importantly, this is a major API break and, as such, would require a major version bump which I don't believe is warranted by this change.